install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -409,9 +409,9 @@ bgpControlPlane:
   # -- SecretsNamespace is the namespace which BGP support will retrieve secrets from.
   secretsNamespace:
     # -- Create secrets namespace for BGP secrets.
-    create: true
+    create: false
     # -- The name of the secret namespace to which Cilium agents are given read access
-    name: cilium-bgp-secrets
+    name: kube-system
```

Comment:
The change makes sense to me. I assume we do not need to care about backward compatibility upon upgrade, as this feature was not yet released in any Cilium release, right?

Reply:
Yes, right